auth-sessions/README.md

```diff
@@ -0,0 +1,66 @@
+# Firebase Admin Node.js Session Management Quickstart
+
+This sample app demonstrates how to use Firebase `httpOnly` session cookies
+with the Firebase Admin SDK session management API.
+
+## Table of Contents
+
+1. [Developer Setup](#developer-setup)
+
+## Developer Setup
+
+### Dependencies
+
+To set up a development environment to build the sample from source, you must
+have the following installed:
+- Node.js (>= 6.0.0)
+- npm (should be included with Node.js)
+
+Download the Firebase source and its dependencies with:
+
+```bash
+git clone https://github.com/firebase/quickstart-nodejs.git
+cd quickstart-nodejs/auth-sessions
+npm install
+```
+
+### Configuring the app
+
+In the Firebase Console, make sure Email/Password and Google providers are enabled.
+
+In the root folder `./auth-sessions`, create a `config.js` file:
+
+```javascript
+var config = {
+  apiKey: "...",
+  authDomain: "...",
+  databaseURL: "...",
+  storageBucket: "...",
+  messagingSenderId: "..."
+};
+module.exports = config;
+```
+Copy and paste the Web snippet code configuration found in the console (either by clicking "Add
```

Comment:
I had to read this a couple of times to get the picture. Consider turning the text in the parentheses into separate sentences or bullet points.

Reply:
Yeah, it was confusing. I have revised this paragraph.